include/exadg/grid/grid.cpp

```diff
@@ -72,18 +72,32 @@ BalancedGranularityPartitionPolicy<dim, spacedim>::partition(
 template<int dim>
 Grid<dim>::Grid(const GridData & data, MPI_Comm const & mpi_comm)
 {
+  typename dealii::parallel::distributed::Triangulation<dim>::Settings distributed_settings;
+
+  if(data.create_coarse_triangulations)
```

Comment:
if you think `create_coarse_triangulations` is a dependent parameter, it should not be part of `GridData`, but instead be passed to the constructor of `Grid` as an argument. When calling the constructor, this parameter has to be correctly deduced from the other `Parameters`. There are examples in ExaDG where we added public functions to `Parameters` in order to obtain certain information already contained in other parameters.